app/components/ui/inline/Button/Button.tsx

```diff
@@ -22,6 +22,7 @@ export const Button = ({
   disabled,
   href,
   mobileFullWidth = true,
+  isVisualOnly = false,
```

Comment:
What about calling this something like `hideFromScreenReader`? "visual only" doesn't necessarily read like an a11y-related property on glance.

Reply:
@rosschapman So in this case, the most accessible thing to do in the `OppEventCard` is just have a `<p>` tag that says "View more ->" but no matter where the user clicks, they are clicking on a link. I don't know if `hideFromScreenReader` conveys that it is ONLY a visual cue, not a button or a link. 

An alternative is to just put the `<p> `tag in the `OppEventCard` component and use a global styling to make it look like a link variant `Button`, rather than have the `Button` component return a `<p>` tag. Thoughts?